docs/victory-voronoi-container/docs.md

```diff
@@ -0,0 +1,132 @@
+# VictoryVoronoiContainer
+
+`VictoryVoronoiContainer` calculates a voronoi diagram based on the data of every child component it
```

Comment:
sorry, one more thing...
It might be more illustrative to say the container "draws" a voronoi diagram, rather than "calculates" - calculates might mean it crunches some numbers and sends them to a callback, but doesn't _display_ anything. Or I could be completely wrong and it doesn't draw anything. In that case, your second sentence is the most important, mentioning mouse position + nearest data point. In fact, if the heart of this component is that, maybe we should rename it to something like "ClosestPointContainer".